### PR TITLE
refs #17044 - update foreman-debug man page for -s option change

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: man/foreman-debug.8.asciidoc [hunk 1]
```diff
@@ -60,7 +60,7 @@ following environment variables can be defined:
  - DIR: Directory to place the tarball in (string)
  - NOGENERIC:  Skip generic info (0 or 1)
  - NOTAR:  Do not create tarballs (0 or 1)
- - MAXLINES: Maximum lines to keep for files (integer)
+ - MAXSIZE: Maximum log set size in bytes (integer)
```
Comment (komidore64) [inline man/foreman-debug.8.asciidoc:63]: Should this say `megabytes` or is `bytes` correct?

Comment (domcleal) [reply man/foreman-debug.8.asciidoc:63]: bytes: https://github.com/theforeman/foreman/blob/develop/script/foreman-debug#L136, https://github.com/theforeman/foreman/blob/develop/script/foreman-debug#L191-L193

Comment (komidore64) [reply man/foreman-debug.8.asciidoc:63]: :+1:

**Comment by lzap** [comment]
Sorry about that, my bad.

**Pull request merged**